Hi @MarcOrfilaCarreras,

The new code is giving below error - 
quilly  | [2023-10-24 10:18:08 +0000] [1] [INFO] Starting gunicorn 21.2.0
quilly  | [2023-10-24 10:18:08 +0000] [1] [ERROR] Retrying in 1 second.
quilly  | [2023-10-24 10:18:09 +0000] [1] [ERROR] Retrying in 1 second.
quilly  | [2023-10-24 10:18:10 +0000] [1] [ERROR] Retrying in 1 second.
quilly  | [2023-10-24 10:18:11 +0000] [1] [ERROR] Retrying in 1 second.
quilly  | [2023-10-24 10:18:12 +0000] [1] [ERROR] Retrying in 1 second.
quilly  | [2023-10-24 10:18:13 +0000] [1] [ERROR] Can't connect to ('0.0.0.0', 80)

Port 80 requires superuser privileges. Please change the port 80 to 8000.
It is working after changing the port.

--- a/Dockerfile
+++ b/Dockerfile
@@ -30,8 +30,8 @@ RUN mkdir -p ./app && \
 USER gunicorn
 
 # Expose the port
-EXPOSE 80
+EXPOSE 8000
 
 # Set the entrypoint and command
 ENTRYPOINT ["gunicorn"]
-CMD ["--bind", "0.0.0.0:80", "entrypoint:app"]
+CMD ["--bind", "0.0.0.0:8000", "entrypoint:app"]

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -9,7 +9,7 @@ services:
           - "GID=1000"
       container_name: quilly
       ports:
-        - 5000:80
+        - 5000:8000
       volumes:
         - ./data:/app/data
       restart: always

